api/app/src/Logger/OpgLineFormatter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Logger;
+
+use Monolog\Formatter\LineFormatter;
+
+class OpgLineFormatter extends LineFormatter
+{
+    public function format(array $record): string
+    {
+        $output = parent::format($record);
+
+        // Split the log message into components
+        preg_match('/^\[(.*?)\] (.*?): (.*)$/', $output, $matches);
+
+        if (4 === count($matches)) {
+            // Format date (light blue)
+            $date = "\033[0;94m".$matches[1]."\033[0m";
+
+            // Format log level
+            $level = $matches[2];
```

Comment:
This whole section should be put under a switch statement

Reply:
yeah was gonna do that originally but needed to use str_contains as the log levels are prefixed so thought if would be easier. However I can do a switch like this I guess :)